src/cm.css

```diff
@@ -170,6 +170,7 @@ a:focus {
   text-shadow: none;
   padding-left: 25px;
   white-space: nowrap;
```

Comment:
`white-space: nowrap` above (twice nonetheless) will set this to not wrap. We just need to remove those lines and add `word-wrap: break-word`

Reply:
oh nice